src/airflow/airqo_etl_utils/airnow_utils.py

```diff
@@ -62,10 +61,9 @@ def extract_bam_data(start_date_time: str, end_date_time: str) -> pd.DataFrame:
 
     @staticmethod
     def process_bam_data(data: pd.DataFrame) -> pd.DataFrame:
-
         air_now_data = []
 
-        devices = AirQoApi().get_tenant_devices(tenant=Tenant.US_EMBASSY)
+        devices = AirQoApi().get_devices(tenant=Tenant.US_EMBASSY)
```

Comment:
Hi @NoahNsimbe please shed more light on this. I do notice that the variable name is still the same (US_EMBASSY). Perhaps the referenced value is different?

Reply:
> Thanks @NoahNsimbe for this much needed PR. Just left a few clarification comments....in addition to the ones below:
> 
> 1. Just noticed that a good number of changes are just new lines/spaces. Could it be due to some style linters?
> 2. As we go about, is it possible to just hardcode tenant to be equal airqo for all your API requestd since **tenant** now just refers to a database name in the backend? AND then just "play" around with the network query parameter since it is the only that references the organisations, going forward?
> 
> Thanks again!

Just noticed that a good number of changes are just new lines/spaces. Could it be due to some style linters? 

`Yeap, Code linters.`

As we go about, is it possible to just hardcode tenant to be equal airqo for all your API requestd since **tenant** now just refers to a database name in the backend? AND then just "play" around with the network query parameter since it is the only that references the organisations, going forward?

Hi @Baalmart 

We already has this hardcoded. Refers to this comment https://github.com/airqo-platform/AirQo-api/pull/1398#discussion_r1139853147

We are also using the netowork parameter. Refers to this comment https://github.com/airqo-platform/AirQo-api/pull/1398#discussion_r1139853419